go/apps/api/routes/v2_identities_update_identity/401_test.go

```diff
@@ -24,12 +24,12 @@ func TestUnauthorized(t *testing.T) {
 	h.Register(route)
 
 	t.Run("missing Authorization header", func(t *testing.T) {
-		identityID := uid.New(uid.IdentityPrefix)
+		externalID := uid.New(uid.IdentityPrefix)
```

Comment:
nit: we shouldn't use an identity prefix here, it might be confusing for future engineers, there's a `uid.TestPrefix` we can use though